pages/blog/january-2021-at-asyncapi.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "January 2021 at AsyncAPI"
+date: 2021-02-11T06:00:00+01:00
+type: Communication
+tags:
+  - status
+cover: /img/posts/january-2021-at-asyncapi/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Dev Comm Keeper
+excerpt: 'I always thought January is a month of a slow start of a new year. I could not have been more wrong. No time to rest, no slow down here.'
+featured: true
+---
+
+A new year always comes with New Year's resolution, right? Folks from Bump.sh came up with the best idea you could get this year. Don't you dare to fail this time :smiley:
+
+<TwitterTweetEmbed
+  tweetId='1346374069877428225'
+  options={{
+    cards: 'hidden',
+    width: 200,
+    maxWidth: 400
```

Comment:
This tweet looks super small (narrow). I found out that changing the CSS makes it a bit better. Have a look.

##### Before:
![Captura de pantalla 2021-02-11 a las 14 14 41](https://user-images.githubusercontent.com/242119/107641120-7ed72580-6c73-11eb-880f-624b4981ed6e.png)

##### After:
![Captura de pantalla 2021-02-11 a las 14 16 30](https://user-images.githubusercontent.com/242119/107641304-bfcf3a00-6c73-11eb-85cf-6db2a803bba3.png)

##### Centering:
![Captura de pantalla 2021-02-11 a las 14 16 10](https://user-images.githubusercontent.com/242119/107641263-b219b480-6c73-11eb-8b9a-e0f72780eb89.png)

Reply:
@fmvilas how about now?

Reply:
Beautiful!